src/cardsdisplay.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-env jquery */
+import {
+  savebutton, taskbutton, selectContainer, myContainer, undoneContainer, selectContainerLabel,
+} from './taskconstants';
+
+const toggleCheckBox = (checkList, card, task) => {
+  if (task.completion) {
+    checkList.checked = true;
+    myContainer.appendChild(card);
+  } else {
+    checkList.checked = false;
+    undoneContainer.appendChild(card);
+  }
+};
+
+const deleteTask = (project, myCard, cardIndex) => {
+  myCard.querySelector('#delete').addEventListener('click', () => {
+    if (window.confirm('Are you sure you want to delete this task?')) {
```

Comment:
- [ ] Do not use Window.alert() or Window.confirm(). It's impossible to style, stop code execution and display differently on different browsers. Use custom modal instead.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/alert_requirement.png)